MetaMask/metamask-extension
# Disable LavaMoat for `start:test` command

The `start:test` command no longer uses LavaMoat to run the build process. `start:test` is meant for local development, so it should have the same behavior as `start` in this respect, and `start` also does not use LavaMoat.

This significantly improves build times for local e2e development.

## Manual Testing Steps

* Run `start:test`
* See that it builds faster, and there is no console output from LavaMoat.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
